### PR TITLE
processor,pkg(ticdc): add Span and package spanz

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

* Add protobuf message `Span`, it will replace table ID in TiCDC code base, and will become the ID of table pipeline.
* Add scaffolds around `Span`, they are dead code for now, will be used in later PRs.
  *  A ordered map of `Span`.
  * A set of `Span`.

Issue Number: ref #7720

#### Tests 

 - Unit test

##### Will it cause performance regression or break compatibility?

No.

##### Do you need to update user documentation, design documentation or monitoring documentation?

No.

### Release note 

```release-note
None
```

### CODE REVIEW

File: pkg/spanz/cov.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 PingCAP, Inc.
```
Comment (3AceShowHand) [inline pkg/spanz/cov.go:1]: what does the file name `cov` mean ?

**Review by 3AceShowHand — APPROVED** [review]

File: pkg/spanz/cov.go (hunk 1 above)
Comment (overvenus) [reply pkg/spanz/cov.go:1]: It means convert. Will update the file name.

**Review by asddongmen — APPROVED** [review]

**Pull request merged**